stable/elasticsearch-exporter/templates/prometheusrule.yaml

```diff
@@ -0,0 +1,50 @@
+{{- if .Values.prometheusRule.enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ template "elasticsearch-exporter.fullname" . }}
+  labels:
+    chart: {{ template "elasticsearch-exporter.chart" . }}
+    app: {{ template "elasticsearch-exporter.name" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+    {{- if .Values.prometheusRule.labels }}
+    {{- toYaml .Values.prometheusRule.labels | nindent 4 }}
+    {{- end }}
+  namespace: {{ .Values.prometheusRule.prometheusNamespace }}
+spec:
+  groups:
+  - name: elasticsearch_{{ .Release.Name }}
+    rules:
+    - record: elasticsearch_filesystem_data_used_percent
+      expr: 100 * (elasticsearch_filesystem_data_size_bytes{job="{{ .Release.Name }}"} - elasticsearch_filesystem_data_free_bytes{job="{{ .Release.Name }}"})
```

Comment:
I wonder how we can even better target the ES cluster. Should we add namespace as well? Should we allow custom filter labels as parameter? I tend to prefer the first solution as it is automated. A similar job could in theory exist in another namespace.